20231019/examples/valid/about.json

```diff
@@ -0,0 +1,23 @@
+{
+  "@context": [
+    "https://w3id.org/kim/amb/context.jsonld",
+    {
+      "@language": "de"
+    }
+  ],
+  "name": "Arbeitsblatt - Mon avenir - Französisch - tutory.de",
+  "id": "https://www.tutory.de/w/fbbadf1a",
+  "type": ["LearningResource"],
+  "about": [
+    {
+      "id": "http://w3id.org/kim/schulfaecher/s1009",
+      "prefLabel": {
+        "de": "Französisch"
+      },
+      "type": "Concept",
+      "inScheme": {
+        "id": "http://w3id.org/kim/schulfaecher/"
+      }
```

Comment:
`inScheme` entfernen, da es auch in der Spec nicht mehr erwähnt wird?

Reply:
Fixed with [39adb10](https://github.com/dini-ag-kim/amb/pull/256/commits/39adb101cd746293be87af597cd4f2262f3e622e)